internal/observability/event/context.go

```diff
@@ -119,28 +118,26 @@ func WriteError(ctx context.Context, caller Op, e error, opt ...Option) {
 		eventer = SysEventer()
 		if eventer == nil {
 			logger := hclog.New(nil)
-			logger.Error(fmt.Sprintf("%s: no eventer available to write error: %s", op, e))
+			logger.Error(fmt.Sprintf("%s: no eventer available to write error: %v", op, e))
 			return
 		}
 	}
 	opts := getOpts(opt...)
 	if opts.withRequestInfo == nil {
 		var err error
 		if opt, err = addCtxOptions(ctx, opt...); err != nil {
-			eventer.logger.Error(errors.Wrap(err, op).Error())
-			eventer.logger.Error(fmt.Sprintf("%s: unable to process context options to write error: %s", op, e))
+			eventer.logger.Error(fmt.Sprintf("%s: %v", op, err))
+			eventer.logger.Error(fmt.Sprintf("%s: unable to process context options to write error: %v", op, e))
 			return
 		}
 	}
 	ev, err := newError(caller, e, opt...)
 	if err != nil {
-		eventer.logger.Error(errors.Wrap(err, op).Error())
-		eventer.logger.Error(fmt.Sprintf("%s: unable to create new error to write error: %s", op, e))
+		eventer.logger.Error(fmt.Sprintf("%s: unable to create new error to write error: %v", op, e))
```

Comment:
I think, you've dropped the err from newError(...) here.  perhaps we still want to log that as well.